js/core/utils/size.js

```diff
@@ -140,24 +144,55 @@ export const getVisibleHeight = function(element) {
     return 0;
 };
 
-export const getWidth = (el) => elementSize(el, 'width');
-export const setWidth = (el, value) => elementSize(el, 'width', value);
-export const getHeight = (el) => elementSize(el, 'height');
-export const setHeight = (el, value) => elementSize(el, 'height', value);
-export const getOuterWidth = (el) => elementSize(el, 'outerWidth');
-export const setOuterWidth = (el, value) => elementSize(el, 'outerWidth', value);
-export const getOuterHeight = (el) => elementSize(el, 'outerHeight');
-export const setOuterHeight = (el, value) => elementSize(el, 'outerHeight', value);
-export const getInnerWidth = (el) => elementSize(el, 'innerWidth');
-export const setInnerWidth = (el, value) => elementSize(el, 'innerWidth', value);
-export const getInnerHeight = (el) => elementSize(el, 'innerHeight');
-export const setInnerHeight = (el, value) => elementSize(el, 'innerHeight', value);
+export const commonCallbacks = {
```

Comment:
Don't we need to wrap it into the DEBUG / ENDDEBUG?

Reply:
No, in this case we need to create a separate implementation of `getWidth` and other methods. It is impossible since `#debug` and `#enddebug` directives are not enabled now, moreover, I don't see if we have an `#else` directive

Reply:
rename to `implementationsMap`